docs/user/gcp/install.md

```diff
@@ -1,10 +1,8 @@
 # Cluster Installation
 
 At this point, you are ready to perform the OpenShift installation.
-You have two choices for installing your cluster on GCP, installer-provided infrastructure or user-provided infrastructure.
-See below for an example of an installer-provided infrastructure install.
+You have two choices for installing your cluster on GCP, installer-provided infrastructure or user-provided infrastructure.  In both cases you should make sure your project has [sufficient quota](quota.md).  To see a guided example of a user-provided infrastructure install, please see [Install: User-Provided Infrastructure](install_upi.md)  See below for an example of an installer-provided infrastructure install.
```

Comment:
I think it is important that the docs for various platforms stay consistent for manageability.

The limits are mentioned here on AWS and Azure, https://github.com/openshift/installer/blame/master/docs/user/aws/README.md#L10 and I think that would be the best place to keep this information.

Reply:
Sounds good to me.  To be clear, are you suggesting I create a new shared limits/quota document and then link all the cloud providers to their specific section?

Reply:
Actually, looking back at this, I'm guessing your saying just name the file limits.md and structure the document in a similar way